posthog/tasks/process_event.py

```diff
@@ -194,6 +195,11 @@ def process_event(
         if data.get("$set"):
             _update_person_properties(team_id=team_id, distinct_id=distinct_id, properties=data["$set"])
 
+    if not Event.objects.filter(team_id=team_id).exists():
+        # Capture first event for the team
+        for user in Team.objects.get(pk=team_id).users.all():
+            posthoganalytics.capture(user.distinct_id, "first event ingested")
```

Comment:
This will result in multiple identical `first event ingested` events per user when we make it possible for users to belong to multiple teams.